ydb/docs/en/core/administration/upgrade.md

```diff
@@ -24,20 +23,20 @@ All minor versions within a major version are compatible for updates. Major vers
 * X.Y.* → X.Y+2.*: Update is impossible, major versions are inconsistent.
 * X.Y.* → X.Y-2.*: Update is impossible, major versions are inconsistent.
 
-A list of available versions can be found on the [download page](https://ydb.tech/en/docs/downloads/).
-
-{% note info %}
+A list of available versions can be found on the [download page](https://ydb.tech/en/docs/downloads/). The YDB release policy is described in more details in the [Release management](../development/manage-releases.md) article of the YDB development documentation.
```

Comment:
Unfortunately, english version doesn't have a notification about a danger of downgrading for 2 major versions.